src/Fixer/Import/OrderedImportsFixer.php

```diff
@@ -47,8 +47,9 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
 
         $usesOrder = array();
         foreach ($namespacesImports as $uses) {
-            $usesOrder = array_replace($usesOrder, $this->getNewOrder(array_reverse($uses), $tokens));
+            $usesOrder[] = $this->getNewOrder(array_reverse($uses), $tokens);
         }
+        $usesOrder = call_user_func_array('array_replace', $usesOrder);
```

Comment:
yeah, it's faster indeed, i just hate calling function by strings... harder to refactor and analyse the code... (like is the function called anywhere)